clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -208,11 +209,12 @@ private void handleCatalogCommand() {
     FullName name = new FullName(line);
     String metalake = name.getMetalakeName();
     String outputFormat = line.getOptionValue(GravitinoOptions.OUTPUT);
+    ListOptions listOptions = new ListOptions(outputFormat);
 
     Command.setAuthenticationMode(auth, userName);
 
     if (CommandActions.LIST.equals(command)) {
-      newListCatalogs(url, ignore, metalake).handle();
+      newListCatalogs(url, ignore, metalake, listOptions).handle();
```

Comment:
The order of the parameters should be the same, other calls have listOptions before metlake.

Reply:
Update to `newListCatalogs(url, ignore, outputFormat, metalake).handle();`